android/app/src/main/AndroidManifest.xml

```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="utf-8"?>
-<manifest xmlns:android="http://schemas.android.com/apk/res/android">
+<manifest xmlns:android="http://schemas.android.com/apk/res/android" xmlns:tools="http://schemas.android.com/tools">
```

Comment:
1) Why it wasn't auto generated before when we use tools?
2) Can you add a note in tools that we are targeting Android 12 and above? not sure how clear  "s" is for all of us.

Reply:
Not sure why it needs to be clear? This file likely won't change, and I don't see a benefit of adding more information to it - it is Android specific configuration.

Motivation for why this was added is here(plugin we are using for Bluetooth connection): https://github.com/capacitor-community/bluetooth-le?tab=readme-ov-file#optional-android-12-bluetooth-permissions 

And since the schema for `tools` were missing, and the build failed, I added the necessary schema.